internal/cobraprompt/prompt.go

```diff
@@ -172,6 +172,7 @@ func (co CobraPrompt) Run(ctx context.Context, root *cobra.Command, cnfg *hazelc
 			root.SetFlagErrorFunc(func(_ *cobra.Command, err error) error {
 				return hzcerrors.FlagError(err)
 			})
+			os.Args = promptArgs
```

Comment:
Changing `os.Args` seems strange to me. Why is that required?

Reply:
Explicit parsing takes place in put-all for example. There is a need for assigning given arguments after each interactive interaction.